openedx/core/djangoapps/django_comment_common/comment_client/utils.py

```diff
@@ -204,3 +218,46 @@ def check_forum_heartbeat():
             return 'forum', False, res.get('check', 'Forum heartbeat failed')
     except Exception as fail:
         return 'forum', False, str(fail)
+
+
+def check_service_enabled():
```

Comment:
What is this for? The fact that we have to repeatedly call `check_service_enabled()` is a red flag, and we should get rid of that function.

Reply:
I think we can remove this function/code as we have already added `ENABLE_DISCUSSION_SERVICE=true` in our forum_v2 plugin.